21-web-game.js

```diff
@@ -0,0 +1,203 @@
+            let img = document.createElement("img");
+            img.src = data.data.cards[0]["image"];
+            let faceDown = document.createElement("img");
+            faceDown.src = "https://raw.githubusercontent.com/nk1tz/vanilla-js-blackjack/master/card.png";
+            val.appendChild(img);
+            val.appendChild(faceDown);
+        } else {
+            urls.forEach(card => {
+                let img = document.createElement("img");
+                img.src = card;
+                val.appendChild(img);
+            })
+            for(let i = 0; i < data.data.cards.length; i++){
+                let draw = document.createElement("img");
+                draw.src = data.data.cards[i]["image"];
+                val.appendChild(draw);
+            }
+        }
+    }
+
+    const displayCards = (data, hand) => {
+        let val = ""
+        hand.id === "playerHand" ? val = "playerImgs" : val = "dealerImgs"
+        let imgDiv = document.querySelector("#" + val)
+        if(val === "dealerImgs"){
+            displayDealerCards(data, val);
+        } else {
+            for(let i = 0; i < data.data.cards.length; i++){
+                let img = document.createElement("img");
+                img.src = data.data.cards[i]["image"];
+                imgDiv.appendChild(img);
+            }
+        }
+    }
+
+    const checkPlayerScore = (hand, score) => {
+        let pScore = document.querySelector("#playerScore");
+        let dScore = document.querySelector("#dealerScore");
+        if(hand.id === "playerHand"){               
+            buttonsContainer.style.display = "block";
+            pScore.innerText = `Score: ${score}`;
+            pScore.value = score;
+        } else {
+            dScore.innerText = `Score: ${score}`;
+            dScore.value = score;
+            dScore.style.display = "none";
+        }
+    }
+
+    const endGame = async () => {
+        let playerScore = document.querySelector("#playerScore").value;
+        let dealerScore = document.querySelector("#dealerScore").value;
+        let message = document.createElement("h2");
+        let buttons = document.querySelector("#playerButtons");
+        buttons.innerHTML = "";
+        while(dealerScore < 17){
+            dealerScore = await draw(deck_id, dealerHand, dealerScore);
+        }
+        if(dealerScore > 21) {
+            message.innerText = "Dealer busts!! You win!"
+            reload();
+        }
+        else if(dealerScore === playerScore) {
+            message.innerText = "Draw! Its a tie!";
+            reload();
+        } else if (playerScore > dealerScore){
+            message.innerText = "Player beats the dealer! You win!";
+            reload();
+        } else {
+            message.innerText = "Dealer beats the player! You lose!";
+            reload();
+        }
+        document.querySelector("#dealerScore").style.display = "block";
+        buttons.appendChild(message);
+    }
+
+    const reload = () => {
```

Comment:
Just some words of advice for the future and your future projects. Reloading the page is frowned upon. Its usually better to figure out a way to reset your divs and logic